mrp_unbuild_cust_alu_analytics/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_mrp_unbuild_process_type,access_mrp_unbuild_process_type,model_mrp_unbuild_process_type,base.group_user,1,1,1,1
```

Comment:
Should any user create and delete process types? cc @ChristianSantamaria

Reply:
They can´t, limited process type and incidence type to mrp groups